pkg/constants/constants.go

```diff
@@ -0,0 +1,13 @@
+package constants
+
+const (
+	EnvBuildId    = "BUILD_ID"
+	EnvCommitSha  = "COMMIT_SHA"
+	EnvTagVersion = "TAG_VERSION"
```

Comment:
Hasn't this been TAGGED_VERSIO before? not sure if this was changed on purpose

Reply:
yeah, I was thinking the same thing, changed it to `TAGGED_VERSION`
the proposal had used `tag_version` in the proto, but I don't think it was for a specific reason